api/lib/infrastructure/events/EventDispatcherLogger.js

```diff
@@ -0,0 +1,47 @@
+class EventDispatcherLogger {
+  constructor(monitoringTools, settings) {
+    this._monitoringTools = monitoringTools;
+    this._settings = settings;
+  }
+  onEventDispatchStarted(event, eventHandlerName) {
+    if (this._settings.logging.enableLogStartingEventDispatch) {
```

Comment:
C'est too much de faire la super sécurité suivante ? :
`this._settings?.logging?.enableLogStartingEventDispatch`

Reply:
Non ça peut être un peu plus robuste en effet